services/graph/apps/api/test/setup/package.json

```diff
@@ -1,28 +1,28 @@
 {
-    "name": "setup",
-    "version": "1.0.0",
-    "description": "",
-    "main": "index.ts",
-    "scripts": {
-        "main": "tsx index.ts",
-        "test": "echo \"Error: no test specified\" && exit 1"
-    },
-    "type": "module",
-    "author": "",
-    "license": "UNLICENSED",
-    "dependencies": {
-        "@amplica-labs/frequency-scenario-template": "^1.1.4",
-        "@dsnp/frequency-schemas": "^1.1.0",
-        "@dsnp/graph-sdk": "^1.1.3",
-        "@frequency-chain/api-augment": "^1.10.0",
-        "@polkadot/api": "^10.9.1",
-        "@polkadot/keyring": "^12.4.2",
-        "@polkadot/types": "^10.11.2",
-        "@polkadot/util": "^12.4.2",
-        "@polkadot/util-crypto": "^12.6.2",
-        "loglevel": "^1.9.1"
-    },
-    "devDependencies": {
-        "tsx": "^4.15.7"
-    }
+  "name": "setup",
```

Comment:
confirming `format` has been run

Reply:
This is actually from npm formatting it I believe. That said, I ran `make format` and it had formatting changes elsewhere, so I'll push that up as well.